CREATE_ERROR_MESSAGE = 'INTERNAL ERROR: COULD NOT INSERT THE OBJECT IN THE DATABASE'
OBJECT_NOT_FOUND_ERROR_MESSAGE = 'OBJECT_NOT_FOUND'

GRAPH_ERRORS = {
    "node_not_found": "El nodo '{node_id}' no existe en el grafo '{graph_id}'",
    "no_path": "No hay ruta entre {start} y {end}"
}